### ISSUE TITLE
[Runtime error]: Cannot read property '0' of undefined

### ISSUE THREAD

**Issue by gundamkid** [FormidableLabs/urql-devtools#339]
## About
A runtime error occurred.

## Stack trace

```
TypeError: Cannot read property '0' of undefined
    at chrome-extension://mcfphkbpmkbeofnkjehahlmidmceblmm/panel.js:8:38500
    at Ct (chrome-extension://mcfphkbpmkbeofnkjehahlmidmceblmm/panel.js:2:1202796)
    at e.generateAndInjectStyles (chrome-extension://mcfphkbpmkbeofnkjehahlmidmceblmm/panel.js:2:1204754)
    at chrome-extension://mcfphkbpmkbeofnkjehahlmidmceblmm/panel.js:2:1206085
    at Gt (chrome-extension://mcfphkbpmkbeofnkjehahlmidmceblmm/panel.js:2:1206152)
    at TimelineSourceIcon (chrome-extension://mcfphkbpmkbeofnkjehahlmidmceblmm/panel.js:2:1207265)
    at Qo (chrome-extension://mcfphkbpmkbeofnkjehahlmidmceblmm/panel.js:2:1043232)
    at Ma (chrome-extension://mcfphkbpmkbeofnkjehahlmidmceblmm/panel.js:2:1051184)
    at xs (chrome-extension://mcfphkbpmkbeofnkjehahlmidmceblmm/panel.js:2:1091530)
    at yu (chrome-extension://mcfphkbpmkbeofnkjehahlmidmceblmm/panel.js:2:1082024)
```

**Comment by andyrichardson** [comment]
Thanks for the report @gundamkid 

We're going to need to have some more reproduction information before we can look into this.

**Comment by andyrichardson** [comment]
Closing this as it is likely a duplicate of #330
